src/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Checkbox from './checkbox';
+
+export default {
+  component: Checkbox,
+  title: 'Components/Checkbox',
```

Comment:
We can add an "[Action](https://storybook.js.org/docs/react/essentials/actions)" here to showcase the `onChange` property. With the below addition, Storybook will automatically print something in the Actions tab of the component by attaching a special Action callback to the `onChange` property.
```suggestion
  title: 'Components/Checkbox',
  argTypes: {
    onChange: { action: 'onChange' },
  },
```